src/main/java/com/projecturanus/betterp2p/util/GlStateManager.java

```diff
@@ -184,7 +184,7 @@ public static void glBlendEquation(int blendEquation) {
     public static void enableOutlineMode(int color) {
         BUF_FLOAT_4.put(0, (float) (color >> 16 & 255) / 255.0F);
         BUF_FLOAT_4.put(1, (float) (color >> 8 & 255) / 255.0F);
-        BUF_FLOAT_4.put(2, (float) (color >> 0 & 255) / 255.0F);
+        BUF_FLOAT_4.put(2, (float) (color & 255) / 255.0F);
```

Comment:
Leave the 0, as it conveys more clearly that we are extract the bottom byte

Reply:
Thank you! I addressed it in https://github.com/GTNewHorizons/BetterP2P/pull/30/commits/d7d72bf230bac8f5675c3cc88f83960d34b1d9f4